msal/build/VerifyAssemblySigning.proj

```diff
@@ -1,50 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We don't need to do assembly signing verification - the signing is done as part of the VSTS sign task

Reply:
I believe that Duncan had had it because at that time the code sign-in task could be successful but the signing not be done (in some weird cases)